docs/Design.md

```
@@ -154,13 +154,13 @@ An IFTTT-compatible service can implement:
 
 ### Production server
 
-The production server is online at: https://orcanodemonitor2.azurewebsites.net/
+The production server is online at: https://orcanodemonitor.azurewebsites.net/
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)